internal/common/utils.go

```diff
@@ -173,6 +173,8 @@ var allowedFunctions = map[string]struct{}{
 	"toStartOfDay":         {},
 	"toDate":               {},
 	"concat":               {},
+	"in":                   {},
+	"IN":                   {},
```

Comment:
why adding both cases? maybe then it's better to change the logic to actually always lowercase?